generator/src/main/java/org/corfudb/generator/LongevityApp.java

```diff
@@ -114,17 +114,17 @@ private void waitForAppToFinish() {
             taskProducer.shutdownNow();
             checkpointer.shutdownNow();
 
-            boolean checkpointHasFinished = false;
-            int exitStatus;
-            try {
-                checkpointHasFinished = checkpointer.awaitTermination(APPLICATION_TIMEOUT_IN_MS, TimeUnit.MILLISECONDS);
-            } catch (InterruptedException e) {
-                Thread.currentThread().interrupt();
-                exitStatus = 1;
-            }
-
-            exitStatus = checkpointHasFinished ? 0 : 1;
-            System.exit(exitStatus);
+//            boolean checkpointHasFinished = false;
```

Comment:
Is there a reason this is part of this PR?

Reply:
oops.. will remove, this was part of another test I was doing... plus I'll add the javadocs